man/special-symbols.Rd

```diff
@@ -66,5 +68,25 @@ DT[{cat(sprintf('in i, .N is \%d\n', .N)); a < .N/2},
 # .I can be different in j and by, enabling rowwise operations in by
 DT[, .(.I, min(.SD[,-1]))]
 DT[, .(min(.SD[,-1])), by=.I]
+
+#These examples illustrate the importance of using copy(.BY) when capturing .BY values in a list outside the data.table context to avoid unexpected behavior.
+dt <- data.table(l = sample(letters[1:5], 100, replace = TRUE), b = rnorm(100))
```

Comment:
hi, this is a good start, but how about a simpler example, with fewer than 100 rows?
how about 4 rows, data.table(letter=c('a','a','b','b'),number=1:4)

Reply:
done :)